Jamulus.pro

```diff
@@ -107,7 +107,7 @@ win32 {
         HEADERS += linux/sound.h
```

Comment:
Should line 104 stop the build?  It's not going to work, after all...

Reply:
I'd say yes.

Reply:
Mm, I'll do that in another PR.

Reply:
If I remember...